### PR TITLE
Consolidate Logos and update README header

### PR DESCRIPTION
**Pull request by silverwind**
- Remove unused `logo-lg.png`, `logo-sm.png` and `logo-192.png`.
- Consolidate `favicon.svg` and `logo.svg` to just `logo.svg`.
- Remove Safari Mask icon, `favicon.png` works just fine since Safari 12.
- Remove Fluid Icon. It only served Firefox and SVG works just fine there.
- Update customization instructions.
- Update README.md to use SVG icon, increase logo size and center it and badges.

### CODE REVIEW

File: README_ZH.md [hunk 1]
```diff
@@ -14,6 +17,8 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 [![Crowdin](https://badges.crowdin.net/gitea/localized.svg)](https://crowdin.com/project/gitea)
 
+[View the english version of this document](README.md)
```
Comment (silverwind) [inline README_ZH.md]: If someone can translate this, it'd be appreciated.

File: README_ZH.md [hunk 2]
```diff
@@ -1,18 +1,52 @@
...
+  <a href="https://crowdin.com/project/gitea" title="Crowdin">
+    <img src="https://badges.crowdin.net/gitea/localized.svg">
+  </a>
+  <a href="https://www.tickgit.com/browse?repo=github.com/go-gitea/gitea" title="TODOs">
+    <img src="https://badgen.net/https/api.tickgit.com/badgen/github.com/go-gitea/gitea">
+  </a>
+</p>
+
+<p align="center">
+  <a href="README_ZH.md">View the english version of this document</a>
```
Comment (lunny) [inline README_ZH.md]:
```suggestion
  <a href="README.md">View the English version of this document</a>
```

**Review by lunny — APPROVED** [review]

File: README_ZH.md [hunk 3]
```diff
@@ -1,18 +1,52 @@
...
+  <a href="https://crowdin.com/project/gitea" title="Crowdin">
+    <img src="https://badges.crowdin.net/gitea/localized.svg">
+  </a>
+  <a href="https://www.tickgit.com/browse?repo=github.com/go-gitea/gitea" title="TODOs">
+    <img src="https://badgen.net/https/api.tickgit.com/badgen/github.com/go-gitea/gitea">
+  </a>
+</p>
+
+<p align="center">
+  <a href="README.md">View the English version of this document</a>
```
Comment (silverwind) [inline README_ZH.md]:
```suggestion
  <a href="README.md">View the english version of this document</a>
```

**Comment by silverwind** [comment]
Note that some sources say that a 192px icon is required for the Homescreen icon on Android but I just tested with 512px only and it seems to work just as well, so I think we can keep it to the one size in the manifest.

**Comment by silverwind** [comment]
Also tested with mobile Firefox and it reveals what is most likely a Firefox bug, see https://github.com/mozilla-mobile/fenix/issues/17215.

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
🚀

**Pull request merged**